ts/session/types/PubKey.ts

```diff
@@ -1,10 +1,14 @@
 export class PubKey {
   public static readonly PUBKEY_LEN = 66;
+  public static readonly MOBILE_GROUP_PUBKEY_LEN = 32;
```

Comment:
i don't think we should mix PubKey and GroupId together, they should be different types